### PR TITLE
all: fix lint

### PR DESCRIPTION
**Pull request by voidzcy**

### CODE REVIEW

File: api/src/test/java/io/grpc/LoadBalancerTest.java [hunk 1]
```diff
@@ -127,6 +124,7 @@ public void pickResult_equals() {
 
   @Deprecated
   @Test
+  @SuppressWarnings("deprecation")
```
Comment (ran-su) [inline api/src/test/java/io/grpc/LoadBalancerTest.java]: I thought we should have comments on SuppressWarnings to let people know why you are suppressing them and why it is safe.

Comment (voidzcy) [reply api/src/test/java/io/grpc/LoadBalancerTest.java]: These are tests. And only for getting rid of lint warnings. Whatever, I will just remove them.

Comment (dapengzhang0) [reply api/src/test/java/io/grpc/LoadBalancerTest.java]: @ran-su When you add `@Deprecated` to the method on purpose (e.g. these tests will be removed once the deprecated API is removed), it does not need `@SuppressWarnings("deprecation")` anymore.

**Comment by voidzcy** [comment]
There is another one..

**Pull request merged**